src/docker-utils.coffee

```diff
@@ -52,9 +52,9 @@ getRepoAndTag = (image) ->
 			registry = ''
 		return { repo: "#{registry}#{imageName}", tag: tagName }
 
-applyDelta = (imgSrc, deltaUrl, { requestTimeout, applyTimeout, retryCount, retryInterval }, onProgress) ->
+applyDelta = (imgSrc, deltaUrl, requestTimeout, applyTimeout, resumeOpts, onProgress) ->
```

Comment:
I'd rather keep the function signature as `(imgSrc, deltaUrl, options, onProgress)`, i.e. making this `(imgSrc, deltaUrl, { requestTimeout, applyTimeout, resumOpts }, onProgress)`, what do you think?
Either that or keep the function definition as before and build `resumeOpts` inside this function.

Reply:
Sounds good -- PR updated.